Makefile

```diff
@@ -58,6 +58,8 @@ db:             ## Reset the database and load fixtures
 db: vendor
 	$(RUN) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MySQL
 	$(RUN) $(CONSOLE) doctrine:database:drop --force --if-exists
+	$(RUN) $(CONSOLE) doctrine:database:create --if-not-exists --env=test_mysql || true
+	$(RUN) $(CONSOLE) doctrine:schema:create --env=test_mysql || true
```

Comment:
Why is it needed here?

**If** you need to create the database for `test_mysql` env, then you need to create it for the `test_sqlite` env.
see https://github.com/EnMarche/en-marche.fr/blob/develop/Makefile#L126